src/index.ts

```diff
@@ -22,13 +22,17 @@ swdcTracker.initialize = async (swdcApiHost: string, namespace: string, appId: s
     const tracker_api_host = result.data.tracker_api
 
     // initialize snowplow tracker
-    const e = emitter(tracker_api_host, 'https', null, 'post', 0)
+    let protocol = "https";
+    if (tracker_api_host.indexOf("http") === 0) {
```

Comment:
```
const protocol = tracker_api_host === "localhost:9090" ? "http" : "https"
```

Reply:
it looks like vscode is sending the full url: https://api.software.com

when we run it in dev mode on our localhost the end point is "http://localhost:5000". why do we want to restrict it to "localhost:9090"?

Reply:
export const api_endpoint = "https://api.software.com";

const result = await swdcTracker.initialize(
      api_endpoint,
      "CodeTime",
      this.pluginParams.plugin_name
    );

Reply:
Let's add the url scheme into the config endpoint.   No need to put the logic in the client.

Reply:
Config JSON will be 
{
  tracker_api: "the-url",
  url_scheme: "https"
}

Reply:
i definitely like configs. what do we do for localhost development? snowplow micro doesn't work if it's https

Reply:
Bo, my apologies, I see what you're saying about 9090. I'll update the init logic to take a url config, thanks.

Reply:
i've reverted it back to use "http" so the url config can go in to a specific task since that logic isn't part of the current issue.

I've also added the request callback to log an error

Reply:
thanks for the feedback. any more thoughts on this? I can build a vsix that we can all test before publishing, and I'd like to get back to that thread to see if we can help the users having issues. thanks again.